### ISSUE TITLE
Using the HuggingFace API

### ISSUE THREAD

**Issue by erlebach** [dottxt-ai/outlines#1280]
Is it possible to create a program that uses a model on HuggingFace together with Outlines via the HuggingFace API (i.e., avoiding download of the model). So far, it appears that the answer is that it is not possible. If it is not possible, could you please explain the difficulty? Is it the impossibility of accessing the output logits of a model running using the API? Thanks.

**Comment by rlouf** [comment]
We would need to integrate the method directly on Hugging Face's servers.
